vllm_ascend/models/deepseek_dbo.py

```diff
@@ -76,7 +76,6 @@
                                               MultiStreamStepMetadata,
                                               make_multistream_metadata_ds)
 from vllm_ascend.ops.fused_moe import AscendFusedMoE
-from vllm_ascend.utils import dispose_tensor
```

Comment:
this function can be removed from utils as well?

Reply:
Not for now. It hasn't been fully decided which `dispose_tensor` calls will be removed. I'll use performance data to justify the necessity of each removal. If it turns out that some `dispose_tensor` calls are still needed, then we can't remove the function from utils.